packages/terra-paginator/tests/jest/ControlledPaginator.test.jsx

```diff
@@ -7,6 +7,24 @@ import ControlledPaginator from '../../src/ControlledPaginator';
 
 describe('Paginator', () => {
   const defaultRender = <ControlledPaginator onPageChange={e => typeof e} selectedPage={1} totalCount={2234} itemCountPerPage={20} />;
+  const renderWithHeadingId = <ControlledPaginator onPageChange={e => typeof e} selectedPage={1} totalCount={2234} itemCountPerPage={20} paginatorHeadingId="paginator_header_id" />;
+  const renderWithAriaLabel = <ControlledPaginator onPageChange={e => typeof e} selectedPage={1} totalCount={2234} itemCountPerPage={20} paginatorLabel="paginator_label" />;
+
+  // Snapshot Tests
```

Comment:
nit: can we just have this comment above the top-most snapshot test rather than above each individual test? Same for the other test files as well

Reply:
Sure thing, updated: https://github.com/cerner/terra-core/pull/3867/commits/7bf4e91d0f599b8a42e6fce81dc07a6da7bceb53